src/main/java/org/tron/core/capsule/TransactionCapsule.java

```diff
@@ -369,9 +367,9 @@ public static long getCpuLimitInTrx(Transaction.Contract contract) {
       Any contractParameter = contract.getParameter();
       switch (contract.getType()) {

```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Remove this conditional structure or edit its code blocks so that they're not all the same. ([squid:S3923](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS3923))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1216&issues=AWTzoQRFJ00hBq3sa8m9&open=AWTzoQRFJ00hBq3sa8m9)

[ISSUE_KEY]: AWTzoQRFJ00hBq3sa8m9